### ISSUE TITLE
[Web] [Accessibility] [11yMAS][Screen Readers][MAS1.1.1] [Alternative text is not provided for Office dev center logo.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#688]
**User Experience:**
User will face difficulty if Alternative text is not provided for Office dev center logo which helps user to know the purpose of the image.
 
**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tools Used: Developer tool

**Attachments:**
![mas1 1 1_alternative text](https://user-images.githubusercontent.com/33054846/32428922-3e895c50-c2ee-11e7-991a-628baea95d9f.png)
![mas1 1 1_alternative text_anchor](https://user-images.githubusercontent.com/33054846/32429008-b3fa3ac2-c2ee-11e7-8e7c-68955b29fe89.png)

**Repro Steps:**
1. Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html .
2. Navigate to Office dev center logo and open dev tool.
3. Verify alternative text is provided or not for “Office dev center logo” . 
 
**Current Behavior:** 
Alternative text is not provided for "Office dev center logo". When user navigates in Item or scan mode screen reader is  announcing it as Image.

**Expected Behavior:**
Alternative text should be provided for logo as ” Office dev center logo” which will provide clear description about the image.

Note:
1. Same issue is repro for the [MAC + safari][ Build info: OS Build: Mac OS Sierra 0.12.6 Browser: Safari 11.0.1] Alternative text is not provided for the Office dev center logo.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}
**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=1.1.1
